### PR TITLE
Make arrows consistent

### PR DESCRIPTION
**Pull request by clienthax**
instead of selectively removing the arrows on the first / last page,
keep the arrows there, but gray them out so the bottom line is constant across all pages

### CODE REVIEW

**Comment by hsyyid** [comment]
This bugs me, thanks for fixing it

File: src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java [hunk 1]
```diff
@@ -160,6 +160,8 @@ protected Text calculateFooter(int currentPage) {
         TextBuilder ret = Texts.builder();
         if (hasPrevious) {
             ret.append(this.prevPageText).append(DIVIDER_TEXT);
+        } else {
+            ret.append(Texts.of("«")).append(DIVIDER_TEXT);
```
Comment (JBYoshi) [inline src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java]: I think these are supposed to be stored in a field set in the constructor (see line 71).

File: src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java [hunk 2]
```diff
@@ -172,6 +174,11 @@ protected Text calculateFooter(int currentPage) {
                 ret.append(DIVIDER_TEXT);
             }
             ret.append(this.nextPageText);
+        } else {
+            if (needsDiv) {
+                ret.append(DIVIDER_TEXT);
+            }
+            ret.append(Texts.of("»"));
```
Comment (JBYoshi) [inline src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java]: Same here.

Comment (clienthax) [inline src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java]: They are private finals declared in the class themselves..

**Pull request closed without merging**